developers/integrations/llm-frameworks/dspy/dspy.md

```diff
@@ -0,0 +1,65 @@
+---
+title: DSPy
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+---
+[DSPy](https://github.com/stanfordnlp/dspy) is a framework for programming language models created by Stanford NLP. DSPy introduces two key concepts: **programming model** and **optimizers**.
+
+- **Programing model**: The programming model lets you define a series of components that make a language model request. Components include input and output fields, task descriptions, and calls to a vector database like Weaviate.
+
+- **Optimizers**: Optimizers compile your DSPy program to tune the language model prompt and/or the weights.
+
+## DSPy and Weaviate
+
+Weaviate is integrated with DSPy through the retriever module! You only need to connect to your Weaviate cluster through [WCD](https://console.weaviate.cloud/) or localhost, and pass in your collection:
+
+```python
+weaviate_client = weaviate.Client("http://localhost:8080") # or pass in your WCD cluster url
+
+retriever_module = WeaviateRM("WeaviateBlogChunk", # collection name
+                    weaviate_client=weaviate_client)
+```
+
+## Our Resources 
+Here are a few resources on using DSPy from the Weaviate team!
+
+The resources are broken into two categories: 
+1. [**Hands on Learning**](#hands-on-learning): Build your technical understanding with end-to-end tutorials.
+
+2. [**Read and Listen**](#read-and-listen): Develop your conceptual understanding of these technologies.
+
+
+### Hands on Learning 
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Getting Started with RAG in DSPy | Learn about the four components needed to build a DSPy program: Installation, settings, datasets with `dspy.Example`, LLM metrics, DSPy programming model, and optimization. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/dspy/1.Getting-Started-with-RAG-in-DSPy.ipynb), [Video](https://youtu.be/CEuUG4Umfxs?si=4Gp8gR9glmoMJNaU) |
```

Comment:
There are more than four things listed. Consider

Learn about DSPy and how to build a program: Installation, settings, datasets, LLM metrics, DSPy programming model, and optimization.